Constitution.txt

```diff
@@ -1,490 +1,264 @@
+3: Congress shall make no law respecting an establishment of religion, or prohibiting the free exercise thereof; or abridging the freedom of speech, or of the press; or the right of the people peaceably to assemble, and to petition the Government for a redress of grievances.
 
-Amendment XXII (Amendment 22 - Presidential Tenure)
-1: No person shall be elected to the office of the President more than twice, and no person who has held the office of President, or acted as President, for more than two years of a term to which some other person was elected President shall be elected to the office of the President more than once. But this article shall not apply to any person holding the office of President when this article was proposed by the Congress, and shall not prevent any person who may be holding the office of President, or acting as President, during the term within which this article becomes operative from holding the office of President or acting as President during the remainder of such term.
+4: The right of the people to keep and bear Arms shall not be infringed.
 
-2: This article shall be inoperative unless it shall have been ratified as an amendment to the Constitution by the legislatures of three-fourths of the several states within seven years from the date of its submission to the states by the Congress.    ratified #22
+5: No Soldier shall, in time of peace, be quartered in any house, without the consent of the Owner, nor in time of war, but in a manner to be prescribed by law; and compensation shall be offered to the Owner for such quartering.
 
-Amendment XXIII (Amendment 23 - Presidential Electors for the District of Columbia)
-1: The District constituting the seat of government of the United States shall appoint in such manner as the Congress may direct: A number of electors of President and Vice President equal to the whole number of Senators and Representatives in Congress to which the District would be entitled if it were a state, but in no event more than the least populous state; they shall be in addition to those appointed by the states, but they shall be considered, for the purposes of the election of President and Vice President, to be electors appointed by a state; and they shall meet in the District and perform such duties as provided by the twelfth article of amendment.
+6: The right of the people to be secure in their persons, houses, papers, other information storage devices, and effects, against unwarranted searches and seizures, shall not be violated, and no Warrants shall issue, but upon probable cause, supported by Oath or affirmation, and particularly describing the place to be searched, and the persons or things to be seized, that are approved and signed by a duly appointed Judge or Justice of the Courts. All Warrants shall be matters of public record, made available to the public on or before the date of search or seizure.
```

Comment:
I add reference to information storage devices, to cover computer drives, phones, and any future technological advancements.